content/docs/2.0/scalers/mysql.md

```diff
@@ -33,11 +33,11 @@ You can authenticate by using connection string or password authentication.
 
 **Connection String Authentication:**
 
-- `connection` - Connection string for MySQL database
+- `connectionFromEnv` - Connection string for MySQL database
 
 **Password Authentication:**
 
-- `connection` - Password for configured user to login to MySQL database
+- `connectionFromEnv` - Password for configured user to login to MySQL database
```

Comment:
is this correct? Because above we use connectionstring (which feels more natural as well)